ts/src/bitmex.ts

```diff
@@ -1949,22 +1949,22 @@ export default class bitmex extends Exchange {
         const isTrailingAmountOrder = trailingAmount !== undefined;
         if (isTrailingAmountOrder) {
             const triggerDirection = this.safeString (params, 'triggerDirection');
-            const triggerAbove = (triggerDirection === 'above');
+            const isAscending = (triggerDirection === 'above') || (triggerDirection === 'rising');
             if ((type === 'limit') || (type === 'market')) {
-                this.checkRequiredArgument ('createOrder', triggerDirection, 'triggerDirection', [ 'above', 'below' ]);
+                this.checkRequiredArgument ('createOrder', triggerDirection, 'triggerDirection', [ 'rising', 'falling' ]);
```

Comment:
if we want to backward maintain above/below values, then this needs to be here:
```suggestion
                this.checkRequiredArgument ('createOrder', triggerDirection, 'triggerDirection', [ 'rising', 'falling', 'above', 'below' ]);
```
but i think we can remove the support of above/below at all, not many users would be using, because they were added relatively recently.